lib/cchain/provider/abci.go

```diff
@@ -709,7 +732,7 @@ func queryLatestAttestation(ctx context.Context, cl atypes.QueryClient, chainVer
 			ChainId:   chainVer.ID,
 			ConfLevel: uint32(chainVer.ConfLevel),
 		})
-	if errors.Is(err, sdkerrors.ErrKeyNotFound) {
+	if errors.Is(err, sdkerrors.ErrKeyNotFound) || status.Code(err) == codes.NotFound {
 		return xchain.Attestation{}, false, nil
 	} else if err != nil {
 		return xchain.Attestation{}, false, err
```

Comment:
the function bellow: `attsFromAtHeight` should that also check for `status.Code(err) == codes.NotFound` and return nil instead of an error?

Reply:
No, I don't think so, since the function returns an empty list of nothing is found.